tests/integration/remote/util.py

```diff
@@ -48,11 +48,22 @@ def get_gas_fees_for_remote() -> tuple:
 
     if not gas_resp or gas_resp.status_code != 200:
         print("Invalid response from Polygon gas station. Retry with brownie values...")
+        if chain.id == 80001:
+            return chain.priority_fee, chain.base_fee + 2 * chain.priority_fee, 4
 
-        return chain.priority_fee, chain.base_fee + 2 * chain.priority_fee
+        return chain.priority_fee, chain.base_fee + 2 * chain.priority_fee, 20
 
-    return Web3.toWei(gas_resp.json()["fast"]["maxPriorityFee"], "gwei"), Web3.toWei(
-        gas_resp.json()["fast"]["maxFee"], "gwei"
+    if chain.id == 80001:
+        return (
+            Web3.toWei(gas_resp.json()["fast"]["maxPriorityFee"], "gwei"),
+            Web3.toWei(gas_resp.json()["fast"]["maxFee"], "gwei"),
+            4,
+        )
+
+    return (
+        Web3.toWei(gas_resp.json()["fast"]["maxPriorityFee"], "gwei"),
+        Web3.toWei(gas_resp.json()["fast"]["maxFee"], "gwei"),
+        20,
```

Comment:
the only thing that is different in both cases is a number, right? Why not assign that number instead of copy-ing 4 lines of code?

Reply:
Sure, thank you.